Sources/CorePayments/Networking/TrackingEventsAPI.swift

```diff
@@ -4,13 +4,29 @@ import Foundation
 ///
 /// Details on this PayPal API can be found in PPaaS under Infrastructure > Experimentation > Tracking Events > v1.
 class TrackingEventsAPI {
+        
+    // MARK: - Internal Properties
+
+    private var apiClient: APIClient
+    var coreConfig: CoreConfig // exposed for testing
```

Comment:
I think we can swap the order of these and have the private second

Reply:
Done! b0b6d321001d5b8eb36d50f6e6e4b53cba6921a2